packages/popmotion-pose/src/dom/flip.ts

```diff
@@ -111,40 +111,34 @@ const implicitlyFlipPose = (state: PoserState, nextPose: Pose) => {
   // Set transform origins
   elementStyler.set({ originX, originY });
 
-  // Create target values
-  const flipPoseProps: FlipPose = {};
```

Comment:
`nextPose` is already a copy, so it should be safe to just mutate it here https://github.com/Popmotion/popmotion/pull/571/files#diff-6575bee1d0a5453bf870f9c62efe8823R83